cnr.sh

```diff
@@ -1,72 +1,85 @@
 #!/bin/bash
+#set -e
 
-function update_cnr {
-    echo "wip: install/update cnr-cli"
+LATEST="v0.4.0"
 
+function download_or_noop {
+  if [ ! -e "$HELM_PLUGIN_DIR/cnr" ]; then
```

Comment:
Yes, you can rephrase, this was just an example. 
Idea is to explain that something is missing rather than silently download it.